tests/TestCase/Http/Cookie/RequestCookiesTest.php

```diff
@@ -31,9 +31,7 @@ class RequestCookiesTest extends TestCase
     public $request;
 
     /**
-     * setup
-     *
-     * @return null
+     * {@inheritDoc}
```

Comment:
Shouldn't? Could you please comment only if you are sure? ;) First time I hear of that.